missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+ULM_diplomacy = {
+	slot = 1
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+
+
+	ULM_hre_allies = {
+		icon = mission_alliances.png
+		required_missions = { }
+		position = 1
+		
+		trigger = {
+			calc_true_if = { # alejandro = god
+				amount = 2
+				all_countries = {
+					is_part_of_hre = yes
+					allied_to = ROOT
+				}
+			}
+		}
+
+		effect = {
+			prestige = 20
+			add_country_modifier = {
+				name = "regional_diplomacy"
+				duration = 9125 #25 years
+			}
+		}
+	}
+	
+	ULM_imperial_guarantee = {
+	icon = mission_monarch_in_throne_room.png
+	required_missions = { ULM_hre_allies }
+	position = 2
+		
+		trigger = {
+			calc_true_if = { # alejandro = god
+				amount = 1
+				OR = {
+					all_countries = {
+						is_emperor = yes
+						allied_to = ROOT
+					}
+					all_countries = {
+						is_emperor = yes
+						has_opinion = {
+							who = ROOT
+							value = 100
+						}
+						owes_favors = { who = ULM value = 10 }
+					}
+				}
+			}
+		}
+
+		effect = { # replace with event
+			add_dip_power = 100
+			add_prestige = 20
+			add_imperial_influence = 25
+			all_countries = {
+				is_emperor = yes
+				add_splendor = 200
+				create_guarantee = ULM
+			}
+		}
+	}
+
+	ULM_swabians_and_the_swiss = {
+		icon = mission_italian_condotierre.png
+		required_missions = { ULM_imperial_guarantee }
+		position = 4
+		
+		trigger = {
+			army_size_percentage = 0.8
+			any_country = {
+				rival = ROOT
+				OR = {
+					primary_culture = swiss
+					primary_culture = swabian
+				}
+			}
+		}
+
+		effect = {
+			country_event = ULM_zael_event.1
+		}
+	}
+	
+	ULM_free_city_guard = {
+		icon = mission_lanshknecht_soldier.png
+		required_missions = { ULM_swabians_and_the_swiss ULM_defensive_overhaul }
+		position = 6
+		
+		trigger = {
+			army_tradition = 60
+			army_professionalism = 20
+			OR = { 
+				any_ally = {
+					primary_culture = swabian
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_professionalism = 40
+				}
+				any_ally = {
+					primary_culture = swiss
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_professionalism = 40
```

Comment:
Might make it shorter and write it with `calc_if_true` clause?

Reply:
Im not sure how to do that, but i'll give it a try!